src/Common/src/Interop/Linux/System.Net.Security.Native/Interop.HeimdalNtlm.cs

```diff
@@ -0,0 +1,173 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Security.Authentication;
+using System.Security.Authentication.ExtendedProtection;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class HeimdalNtlm
+    {
+        private const string ClientToServerSigningMagicKey = "session key to client-to-server signing key magic constant\0";
+        private const string ServerToClientSigningMagicKey = "session key to server-to-client signing key magic constant\0";
+        private const string ServerToClientSealingMagicKey = "session key to server-to-client sealing key magic constant\0";
+        private const string ClientToServerSealingMagicKey = "session key to client-to-server sealing key magic constant\0";
```

Comment:
Should these be pre- allocated/encoded, e.g.:

```csharp
private static readonly byte[] s_clientToServerSigningMagicKey = Encoding.UTF8.GetBytes("session key to client-to-server signing key magic constant\0");
private static readonly byte[] s_serverToClientSigningMagicKey = Encoding.UTF8.GetBytes("session key to server-to-client signing key magic constant\0");
private static readonly byte[] s_serverToClientSealingMagicKey = Encoding.UTF8.GetBytes("session key to server-to-client sealing key magic constant\0");
private static readonly byte[] s_clientToServerSealingMagicKey = Encoding.UTF8.GetBytes("session key to client-to-server sealing key magic constant\0");
```

...to avoid having to allocate a new `byte[]` and encode each time they are used?